src/components/FeatureSelector/FeatureMenu.tsx

```diff
@@ -1,35 +1,86 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { FeatureSliderData } from './FeatureSliderData'
-import { Button, Menu, MenuItem } from '@material-ui/core';
+import { Box, Button,  List,  ListItem,  makeStyles, Menu, MenuItem, Paper, Popover } from '@material-ui/core';
+import { Info as InfoIcon } from '@material-ui/icons';
+
 import Options from "./Defaults"
 
+const useStyles = makeStyles(theme => ({
+  popover: {
+    pointerEvents: 'none'
+  },
+  paper: {
+    padding: theme.spacing(1)
+  },
+  button: {
+    margin: theme.spacing(1),
+    whiteSpace: "nowrap"
+  },
+  root: {
+    width: '100%',
+    minWidth: 200
+  }
+}))
+
 export default function FeatureMenu(props: {
     onSelect: (option: FeatureSliderData) => void
     hidden: string[]
   }) {
-    
+    const classes = useStyles()
     const [anchorEl, setAnchorEl] = React.useState(null);
     const handleClose = () => {
       setAnchorEl(null);
     };
     const handleMenuOpen = (event: any) => {
       setAnchorEl(event.currentTarget);
     };
+    const [popupAnchor, setPopupAnchor] = useState(null)
     
     return (
-      <div>
-        <Button aria-controls="feature-menu" aria-haspopup="true" onClick={handleMenuOpen}>
-          Add Feature Filter
-        </Button>
-        <Menu
-          id="feature-menu"
-          anchorEl={anchorEl}
-          keepMounted
-          open={!!anchorEl}
-          onClose={handleClose}
-        >
-          {Options.filter(o => !props.hidden.includes(o.name)).map((o, index) => (<MenuItem key={index} onClick={() => props.onSelect(o)}>{o.name}</MenuItem>))}
-        </Menu>
+      <div className = {classes.root}>
+        <List component={Paper} disablePadding={true}>
```

Comment:
I'm not sure about this paper bit. Nowhere else do we put a paper inside a paper.

It's consuming more screenspace and I'm not sure what the benefit is

Reply:
removed list altogether